CHANGELOG/CHANGELOG-1.9.0.md

```diff
@@ -11,10 +16,26 @@
     - [Features](#features)
     - [Bugs](#bugs-1)
  
+# v1.9.2
+
+## Changelog since v1.9.1
+
+## Changes by Kind
+
+### Bugs
+
+- CSM Operator does not calculate status correctly when a driver is deployed by itself. ([#1130](https://github.com/dell/csm/issues/1130))
+- CSM Operator does not calculate status correctly when a application-mobility is deployed by itself. ([#1133](https://github.com/dell/csm/issues/1133))
+
 # v1.9.1
 
 ## Changelog since v1.9.0
 
+## Known Issues 
+
+- For CSM Operator released in CSM v1.9.1, a plain driver install (no modules) will always be marked as failed in the CSM status even when it succeeds. This is fixed in CSM 1.9.2 (csm-operator v1.4.2). As a workaround, the driver deployment is still usable as long as all the pods are running/healthy.
```

Comment:
Please remove "This is fixed in CSM 1.9.2 (csm-operator v1.4.2).".  The fix is mentioned under 1.9.2 bugs.

Reply:
done